source/dbo/Stored Procedures/Data Migration/uspMigrationAdditionalNotificationsByRegionAndYear.sql

```diff
@@ -14,21 +14,31 @@ AS
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_Hospital] tbh ON tbh.HospitalID = mn.NtbsHospitalId
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_PHEC] tbsp ON tbsp.TB_Service_Code = tbh.TB_Service_Code
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[PHEC] p ON p.PHEC_Code = tbsp.PHEC_Code
+		LEFT JOIN vwNotificationYear ny ON ny.NotificationYear = YEAR(mn.NotificationDate)
 	WHERE 
 	mn.GroupId IS NOT NULL
 	AND p.PHEC_Name = @Region
-	AND mn.NotificationDate >= '2017-01-01')
+	AND ny.Id >= -3
+	AND NOT EXISTS
+		(SELECT LegacyId
+		FROM [$(migration)].[dbo].ImportedNotifications impn
+		WHERE LegacyId = EtsId OR LegacyId = LtbrId))
 
 	SELECT DATEPART(YEAR, NotificationDate) AS 'NotificationYear', p.PHEC_Name AS Region, COUNT(PrimaryNotificationId) AS CountOfRecords  
 	FROM [$(migration)].[dbo].[MergedNotifications] mn 
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_Hospital] tbh ON tbh.HospitalID = mn.NtbsHospitalId
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[TB_Service_to_PHEC] tbsp ON tbsp.TB_Service_Code = tbh.TB_Service_Code
 		LEFT OUTER JOIN  [$(NTBS_R1_Geography_Staging)].[dbo].[PHEC] p ON p.PHEC_Code = tbsp.PHEC_Code
+		LEFT JOIN vwNotificationYear ny ON ny.NotificationYear = YEAR(mn.NotificationDate)
 		INNER JOIN NotificationGroups ng ON ng.GroupId = mn.GroupId
 	WHERE mn.GroupId IS NOT NULL
 	AND 
 	(p.PHEC_Name != @Region
-	OR mn.NotificationDate < '2017-01-01')
+	OR ny.Id >= -3)
```

Comment:
Now, I actually think the existing code is wrong here.  We are going to migrate in any linked record, regardless of whether it's within our time period or not.  So I think we are not displaying them on the report and we should be. I need to look at exactly where this gets called from on the report.

Reply:
I've taken this out for now, if we decide it should be there then we can put it back

Reply:
I have readded this as we realised that it's needed to get the additional notifications. I have also updated the comment